webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -9,33 +9,42 @@ import {
   notificationsDrawer as sessionStorage
 } from '../../../common/sessionStorage';
 import API from '../../../API';
-const getNotificationsInterval = 10000;
+import { isNil } from 'lodash';
+const defaultNotificationsPollingInterval = 10000;
+const notificationsInterval = isNil(process.env.NOTIFICATIONS_POLLING) ?
+  defaultNotificationsPollingInterval :
+  process.env.NOTIFICATIONS_POLLING;
 
 export const getNotifications = url => dispatch => {
   new Promise((resolve, reject) => {
     if (
       document.visibilityState === 'visible' ||
       document.visibilityState === 'prerender'
     ) {
-      API.get(url).then(response => {
-        dispatch({
-          type: NOTIFICATIONS_GET_NOTIFICATIONS,
-          payload: {
-            notifications: response.notifications
-          }
-        });
-        resolve();
-      }, () => reject());
+      API.get(url).then(
+        response => {
```

Comment:
probably just formatting... the relvant change is below with the "if" statement.